specification/common-types/resource-management/v5/mobo.json

```diff
@@ -0,0 +1,37 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "5.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "MoboConfiguration": {
```

Comment:
Sorry, but I don't like this name being an acronym 'mobo'. (Maybe partly just because it sounds too much like it means motherboard or some other uh, negative word..) Here are some alternative suggestions.
-ManagedOnBehalfOfConfiguration
-ManagedOnBehalfConfiguration (better, since 'on behalf of X' has a meaning that doesn't exactly match)
Any other ideas like ManagedByX that might seem less ambiguous in who exactly is going to do the managing? 
Or 'ManagedResourceProfile' if its meant to more flexible than just platform managed?

Reply:
Addressed